examples/simple/simple.tf

```diff
@@ -1,20 +1,20 @@
 provider "aws" {
-  version = "~> 2.42"
+  version = "~> 3.0"
   region  = "us-west-2"
 }
 
 module "acs" {
-  source = "github.com/byu-oit/terraform-aws-acs-info.git?ref=v2.1.0"
+  source = "github.com/byu-oit/terraform-aws-acs-info.git?ref=v3.0.0"
 }
 
 module "rds" {
-  source = "github.com/byu-oit/terraform-aws-rds?ref=v1.1.0"
+  source = "github.com/byu-oit/terraform-aws-rds?ref=v2.0.0"
   //  source                  = "../.."
   identifier              = "example"
   engine                  = "mysql"
   engine_version          = "8.0"
   family                  = "mysql8.0"
-  cloudwatch_logs_exports = ["audit", "error", "general", "slowquery"]
```

Comment:
Did this change for a reason?

Reply:
Yeah when I tried to apply the Terraform in my own tests, it didn't allow `audit`. I assumed that it was something that changed with the provider, it may be nice if someone else could verify it.

Reply:
```Error creating DB Instance: InvalidParameterCombination: You cannot use the log types 'audit' with engine version mysql 8.0.17. For supported log types, see the documentation.```